### PR TITLE
Adds Aim Mode keybinding

### PR DESCRIPTION
**Pull request by Walarks**
## About The Pull Request
Gives a keybinding to Aim Mode button, 6 by default.

## Why It's Good For The Game
I got asked to add one.

## Changelog
:cl:
add: adds aim mode keybinding, 6 by default
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -25. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Pariah919** [comment]
denied

**Review by TiviPlus — APPROVED** [review]

**Comment by InterroLouis** [comment]
Oh no my medic button

**Comment by TGCMbetcher** [comment]
> denied

Why? All it does is stop accidental FF

**Comment by SplinterGP** [comment]
> > denied
> 
> Why? All it does is stop accidental FF

name from before

File: code/datums/actions/item_action.dm [hunk 1]
```diff
@@ -90,6 +90,14 @@
...
 	name = "Take Aim"
 
+/datum/action/item_action/aim_mode/give_action(mob/M)
+	. = ..()
+	RegisterSignal(M, COMSIG_KB_AIMMODE, .proc/action_activate)
+
+/datum/action/item_action/aim_mode/remove_action(mob/M)
+	UnregisterSignal(M, COMSIG_KB_AIMMODE, .proc/action_activate)
+	return ..()
+
```
Comment (SplinterGP) [inline code/datums/actions/item_action.dm:100]: Decide yourself on one or the other

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/datums/actions/item_action.dm (hunk 1 above)
Comment (TiviPlus) [reply code/datums/actions/item_action.dm:100]: Cleared up on discord

**Pull request merged**